src/hooks/imagePicker.ts

```diff
@@ -38,6 +38,7 @@ export const useSelectImage = (
 
     if (!result.cancelled) {
       onChange ? onChange(setImageUri)(result.uri) : setImageUri(result.uri);
+      return result;
```

Comment:
what is `result` in this case? does it still satisfy typescript and match the other return somehow?

Reply:
I am using the `uri` and `type` data from the `result` object. There is `imageUri` returned in `useSelectImage`. But this information is returned to me when I use `useSelectImage`. I need the return in `selectImage`.

Reply:
ok we have called 1 on 1 and discussed the situation. all fine now.